src/io/flutter/NotificationManager.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter;
+
+import javax.annotation.Nullable;
```

Comment:
You'll want to switch this import to `org.jetbrains.annotations.Nullable`.

Reply:
done.